### PR TITLE
Updated init script

### PR DESCRIPTION
**Pull request by steinkohl**
adds the 'owner' role to 'Admins' team members

### CODE REVIEW

File: backend/python_init_script/main.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import os
```
Comment (dominic-heil) [inline backend/python_init_script/main.py:1]: The copyright header is missing.

File: backend/python_init_script/main.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from appwrite.client import Client
+
+# from appwrite.services.database import Database
+from appwrite.services.users import Users
+from appwrite.services.teams import Teams
+
+# from appwrite.services.functions import Functions
```
Comment (dominic-heil) [inline backend/python_init_script/main.py:8]: Can commented imports and statements (below) be removed?

**Review by dominic-heil — COMMENTED** [review]
Tested it on my system with a new appwrite project and everything worked. Only have two small comments. Also, another thing that we might want to change: If there already is an "Admins" team, then a new Admins team is created anyways. Instead, if there already is an Admins team, then I think the previous Admins team should be user or the script should exit.

File: backend/python_init_script/main.py (hunk 1 above)
Comment (steinkohl) [reply backend/python_init_script/main.py:1]: stupid me again

File: backend/python_init_script/main.py (hunk 2 above)
Comment (steinkohl) [reply backend/python_init_script/main.py:8]: yes.

**Comment by steinkohl** [comment]
> Tested it on my system with a new appwrite project and everything worked. Only have two small comments. Also, another thing that we might want to change: If there already is an "Admins" team, then a new Admins team is created anyways. Instead, if there already is an Admins team, then I think the previous Admins team should be used or the script should exit.

I will implement that in the next sprint including handling of user/admin data without exposing to public

**Review by dominic-heil — APPROVED** [review]

**Pull request merged**